doc/user/content/get-started.md

```diff
@@ -109,25 +109,36 @@ For this guide, you'll create 2 clusters, one for ingesting source data and the
 1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the replicas:
 
     ```sql
-    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    SHOW CLUSTER REPLICAS WHERE cluster = 'compute_qck' OR cluster = 'ingest_qck';
     ```
     <p></p>
 
     ```noftm
        cluster   | replica |  size   | ready
     -------------+---------+---------+-------
      compute_qck | r1      | 2xsmall | t
+     ingest_qck  | r1      | 2xsmall | t
     (2 rows)
     ```
 
-    Once the `r1` replica is ready (`ready=t`), move on to the next step.
+    Once both replicas are ready (`ready=t`), move on to the next step.
 
 ## Step 3. Ingest streaming data
 
 Materialize supports streaming data from multiple external sources, including [Kafka](/sql/create-source/kafka/) and [PostgreSQL](/sql/create-source/postgres/). The process for integrating a source typically involves configuring the source's network and creating connection objects in Materialize.
 
 For this guide, you'll use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
 
+1. Aside from clusters and replicas, sources and most other objects in Materialize are [namespaced](/sql/namespaces/) by database and schema, so start by creating a unique schema within the default `materialize` database:
+
+    ```sql
+    CREATE SCHEMA qck;
+    ```
+
+    ```sql
+    SET search_path = qck;
```

Comment:
Counterpoint: I think it's better to rip the complexity bandaid off here because `SET SCHEMA` is a leaky abstraction. Details here: https://github.com/MaterializeInc/materialize/issues/18946#issuecomment-1529215389

Reply:
Leaving this for now. Doesn't feel like something we need to explain in detail in getting started, in any case.